CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 lib_mic_array change log
 ========================
 
+UNRELEASED
+----------
+
+  * ADDED:   Support for XCommon CMake build system on v5.2.0 and above
```

Comment:
It isn't necessary to mention version numbers in the changelog, and this is misleading because v5.2.0 has already been released so it will never have XCommon CMake support.